fuzz/oss-fuzz.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
I am not entirely sure about using others work in your submission, nor am I sure that it is necessary to include this script if it is not going to be used for oss-fuzz anyway (at least for this integration). I will have to check on this.